Comparison/src/cz/fidentis/comparison/procrustes/ProcrustesAnalysis.java

```diff
@@ -542,6 +542,59 @@ public ICPTransformation rotate(ProcrustesAnalysis pa2) {
         
         return new ICPTransformation(null, 1.0f, null, 0.0f, r);
     }
+    
+    /**
+     * Method for PDM computation, works same as normal rotate
+     * http://en.wikipedia.org/wiki/Orthogonal_Procrustes_problem
+     *
+     * @param pa2 another configuration
+     */
+    public ICPTransformation rotatePDM(ProcrustesAnalysis pa2) {

```

Comment:
tato metoda je v podstate to iste ako metoda rotate az na mensie zmeny, je mozne obe metody refaktorovat tak, aby tu nebol duplikovany kod? v pripade ze nastane zmena v metode rotate, bude ju potrebne opravovat na 2 miestach, co je zle